### PR TITLE
Updated Resources lesson

### PR DESCRIPTION
**Pull request by cjhar**
## What does this PR do and why?

Added Cloud9 section to the instructions including upload of the template to S3 bucket

## PR acceptance checklist

This checklist encourages us to confirm any changes have been analyzed to reduce risks in quality, performance, reliability, security, and maintainability.

- [x] Added/Updated documentation if applicable.
- [x] Pre-commit checks passed.
- [x] Lint and Nag checks passed.
- [ ] If releasing a new version, have you bumped the version `make version part=<major|minor|patch>`?

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: content/basics/templates/resources/index.md [hunk 1]
```diff
@@ -116,21 +116,52 @@ To complete this and future labs you will need **Default VPC** in the region you
...
+1. Click **Next**.
+1. Provide a **Stack name**. For example `cfn-workshop-ec2`.
+    + The _Stack name_ identifies the stack. Use a name to help you distinguish the purpose of this stack.
+    + For **Type of EC2 Instance** select your preferred instance size, for example **t2.micro**.
+    + Click **Next**.
+1. You can leave **Configure stack options** default, click **Next**.
+1. On the **Review <stack_name>** page, scroll down to the bottom and click on **Create stack**.
+    ::alert[This will create EC2 instance in your account. To check the cost of the deployed stack, click on **Estimate cost** on the review page.]{type="info"}
+1. You can click the **refresh** button a few times until you see in the status **CREATE_COMPLETE**.
+::::
```
Comment (rezabekf) [inline content/basics/templates/resources/index.md:144]: Please update the instructions to uyse `awscli` to deploy template via CloudFromation. Please see the `content/basics/templates/template-and-stack/index.md` as an example

**Review by rezabekf — CHANGES_REQUESTED** [review]
Please use cfn cli commands rather than S3 bucket upload.

File: content/basics/templates/resources/index.md (hunk 1 above)
Comment (cjhar) [reply content/basics/templates/resources/index.md]: Thanks Franco, updated to new format using CLI and further changes to align with recommended us-east-1 region.

**Review by rezabekf — APPROVED** [review]
LGMT

**Pull request merged**